src/types.ts

```diff
@@ -192,3 +192,10 @@ export const convertContextTypes = function(dbItem: ContextDB): Context {
     translationId: dbItem.translation_id,
   };
 };
+
+export type WordWithTransContextStatusTypes = {
```

Comment:
How about calling this type `Userword`? The service is called `getUserwordsInText` so it would make sense to call the the return type that.